custom_components/kia_uvo/Vehicle.py

```diff
@@ -1,203 +1,195 @@
 import logging
 
-from datetime import datetime
-import re
+from datetime import timedelta, datetime
+import push_receiver
+import random
 import requests
+from urllib.parse import parse_qs, urlparse
+import uuid
+import json
 
-from homeassistant.config_entries import ConfigEntry
-from homeassistant.core import HomeAssistant
-from homeassistant.helpers.dispatcher import (
-    async_dispatcher_connect,
-    async_dispatcher_send,
-)
-from homeassistant.helpers.event import async_call_later
-import homeassistant.util.dt as dt_util
-
-from .const import *
+from .const import DOMAIN, BRANDS, BRAND_HYUNDAI, BRAND_KIA, DATE_FORMAT, VEHICLE_LOCK_ACTION
 from .KiaUvoApiImpl import KiaUvoApiImpl
 from .Token import Token
 
 _LOGGER = logging.getLogger(__name__)
 
-class Vehicle(object):
+
+class KiaUvoApiCA(KiaUvoApiImpl):
```

Comment:
I am lost here, why did you put this class into Vehicle.py ?

Reply:
Looks like a copy and paste mistake.  This was not what was running locally on my side.  Corrected now.